MindOverMapper_Movim/MindOverMapper_Movim/ClientApp/src/components/login/Login.css

```diff
@@ -9,9 +9,6 @@
     display: flex;
     flex-direction: row;
     flex-grow: inherit;
-    height: 101.7vh;
-    overflow-x: hidden;
-    overflow-y: hidden;
```

Comment:
Is this why the login page looks different?

Reply:
Yes, we'll need to make some css updates to get things back in line. That was freezing the page so you couldn't scroll.